src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -9,16 +10,37 @@ import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
 import { Button } from "../Components/Button"
 import { CenteringContainer, Container } from "../Components/Containers"
 import { Divider } from "../Components/Divider"
+import { MarkdownRenderer } from "../Components/MarkdownRenderer"
 import { Timer } from "../Components/Timer"
 
+// import { SelectMaxBid_sale_artwork } from "__generated__/SelectMaxBid_sale_artwork.graphql"
+import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.graphql"
+
+// interface MinimumNextBid {
+//   display: string
+//   cents: number
+// }
+
 interface BidResultProps {
+  sale_artwork: BidResult_sale_artwork
   winning: boolean
   message_header?: string
   message_description_md?: string
+  navigator: NavigatorIOS
 }
 
 export class BidResult extends React.Component<BidResultProps> {
+  onPressBidAgain = () => {
+    // pushing to MaxBidScreen creates a sircular relay reference but this works
+    // TODO: correct the screen transision animation
+    this.props.navigator.popToTop()
+  }
+
   render() {
+    const { live_start_at, end_at } = this.props.sale_artwork.sale
+    // non-live sale doesn't have live_start_at so bidding is open until end time
+    const timeLeftToBid = live_start_at || end_at
+    const timeLeftToBidMillis = Date.parse(timeLeftToBid) - Date.now()
```

Comment:
Okay so I just realized that we're going to have to change this a little bit, to make sure that the `timeLeftToBidMillis` is always accurate, even if the user's device is out of sync with Gravity. I've opened an issue to look into it later: https://artsyproduct.atlassian.net/browse/PURCHASE-133